script/e2e-test.sh

```diff
@@ -395,9 +395,21 @@ kubectl create rolebinding kubeapps-view-user-apprepo-read -n kubeapps-user-name
 kubectl create rolebinding kubeapps-view-user -n kubeapps-user-namespace --clusterrole=edit --serviceaccount kubeapps:kubeapps-view
 ## Create edit user
 kubectl create serviceaccount kubeapps-edit -n kubeapps
+# TODO(minelson): Many of these roles/bindings need to be cleaned up. Some are
+# unnecessary (with chart changes), some should not be created (such as edit
+# here having the edit cluster role in the kubeapps namespace - should just be
+# default).
 kubectl create rolebinding kubeapps-edit -n kubeapps --clusterrole=edit --serviceaccount kubeapps:kubeapps-edit
 kubectl create rolebinding kubeapps-edit -n default --clusterrole=edit --serviceaccount kubeapps:kubeapps-edit
 kubectl create rolebinding kubeapps-repositories-read -n kubeapps --clusterrole kubeapps:kubeapps:apprepositories-read --serviceaccount kubeapps:kubeapps-edit
+# TODO(minelson): Similar to the `global-repos-read` rolebinding that the chart
```

Comment:
Perhaps worth linking the issue Rafa was mentioning (https://github.com/vmware-tanzu/kubeapps/issues/4435) ?

Reply:
Done.